mica/stats/tests/test_acq_stats.py

```diff
@@ -2,11 +2,21 @@
 import tempfile
 import os
 import pytest
+from pathlib import Path
 
 from .. import update_acq_stats as acq_stats
+from .. import acq_stats as read_acq_stats
 
 HAS_OBSPAR_ARCHIVE = os.path.exists(
         acq_stats.mica.archive.obspar.CONFIG['data_root'])
+HAS_ACQ_TABLE = Path(read_acq_stats.TABLE_FILE).exists()
+
+
+@pytest.mark.skipif('not HAS_ACQ_TABLE', reason='Test requires acq stats table')
```

Comment:
Modern preferred idiom is to use the value, not the string repr. So just `skipif(not HAS_ACQ_TABLE, ..)`.